pkg/ledger/nano_S_driver.go

```diff
@@ -219,12 +221,24 @@ func (n *NanoS) SignTxn(txn []byte) (sig [65]byte, err error) {
 }
 
 func (n *NanoS) SignStaking(stake []byte) (sig [65]byte, err error) {
+	buf := bytes.NewBuffer(stake)
 	var resp []byte
 
-	var p1 byte = p1More
-	resp, err = n.Exchange(cmdSignStaking, p1, p2SignHash, stake)
-	if err != nil {
-		return [65]byte{}, err
+	for buf.Len() > 0 {
+		var p1 byte = p1More
+		var p2 byte = p2SignHash
+		if resp == nil {
+			p1 = p1First
+		}
+
+		if buf.Len() < 255 {
+			p2  = p2Finish
+		}
+
+		resp, err = n.Exchange(cmdSignStaking, p1, p2, buf.Next(255))
+		if err != nil {
+			return [65]byte{}, err
```

Comment:
could you please make the 65 be a named constant

Reply:
done